### PR TITLE
Create test for CityService. Fix small bug in forecastPresenterTest.

### PR DESCRIPTION
**Pull request by VdovinN**

### CODE REVIEW

File: app/src/test/java/com/example/nvdovin/weatherapp/domain/service/CityServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    public void setUp() throws Exception {
+        cityService = new CityService(daoSession, context);
+    }
+
+    @Test
+    public void testInsert() throws Exception {
+
+        when(daoSession.getCityDao()).thenReturn(cityDao);
+
+        cityService.insert(city);
```
Comment (victorjimbei) [inline app/src/test/java/com/example/nvdovin/weatherapp/domain/service/CityServiceTest.java:60]: This test is not complete.  Change insert() from cityService to delete(city), and run this test again.

File: app/src/test/java/com/example/nvdovin/weatherapp/domain/service/CityServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        verify(queryBuilder).where(any(WhereCondition.class));
+        verify(queryBuilder).unique();
+    }
+
+    @Test
+    public void testLoadSortedCitiesOrderAsc() throws Exception {
+
+        initBaseCondition();
+
+        doReturn(false).when(sortQueryBuilder).isAscending();
```
Comment (victorjimbei) [inline app/src/test/java/com/example/nvdovin/weatherapp/domain/service/CityServiceTest.java]: why don't keep it consistent, and call here also when from Mokito? 
I mean to have the same as below `when(sortQueryBuilder.isAscending()).thenReturn(false);`

File: app/src/test/java/com/example/nvdovin/weatherapp/domain/service/CityServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+        verifyBaseCondition();
+    }
+
+    @Test
+    public void testLoadSortedCitiesOrderDesc() throws Exception {
+
+        initBaseCondition();
+
+        doReturn(true).when(sortQueryBuilder).isAscending();
```
Comment (victorjimbei) [inline app/src/test/java/com/example/nvdovin/weatherapp/domain/service/CityServiceTest.java]: the same here

**Review by victorjimbei — CHANGES_REQUESTED** [review]

File: app/src/test/java/com/example/nvdovin/weatherapp/domain/service/CityServiceTest.java (hunk 1 above)
Comment (VdovinN) [reply app/src/test/java/com/example/nvdovin/weatherapp/domain/service/CityServiceTest.java:60]: Added necessary condition

File: app/src/test/java/com/example/nvdovin/weatherapp/domain/service/CityServiceTest.java (hunk 2 above)
Comment (VdovinN) [reply app/src/test/java/com/example/nvdovin/weatherapp/domain/service/CityServiceTest.java]: understand

File: app/src/test/java/com/example/nvdovin/weatherapp/domain/service/CityServiceTest.java (hunk 3 above)
Comment (VdovinN) [reply app/src/test/java/com/example/nvdovin/weatherapp/domain/service/CityServiceTest.java]: understand

**Review by victorjimbei — APPROVED** [review]

**Pull request merged**